inspirehep/modules/theme/static/scss/format/brief-results.scss

```diff
@@ -508,4 +508,17 @@ i.fa-chevron-right.facet-slider {
 
 .brief-results-sortby {
 	color: #737373;
+}
+
+div > .modal-dialog {
+    width: 63%;
+}
+
+#textarea-cite {
+    resize: none;
+    width: 100%;
+    min-height: 237px;
+    background-color: #f5f5f5;
```

Comment:
better use variable names here. see for example https://github.com/inspirehep/inspire-next/commit/e31d1bfcc2d7c10906201cdb14715b0ac81f2e75